OpenRA.Mods.Common/Traits/BlocksProjectiles.cs

```diff
@@ -12,6 +12,11 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public interface IBlocksProjectiles
```

Comment:
and a `[RequireExplicitImplementation]` here too if that doesn't require too many followup changes.